### ISSUE TITLE
tb-columns doesn't invoke data function on Table

### ISSUE THREAD

**Issue by ridaamirini** [TerryZ/v-selectpage#22]
Hello,

this doesn't work on tb-columns property:

```Javscript
[
  {
     title: 'Test',
    data(row) {
        return row.firstname + '  ' + row.lastname;
     }
   }
]
```
I think its caused by this [line](https://github.com/TerryZ/v-selectpage/blob/f56965b2afdca3a308de1d6d60d7f8b1596ee293/src/components/Table.vue#L11)

```HTML
<td :key="idx" v-for="(col,idx) in tbColumns" v-html="row[col.data]"></td>
```

It should also call `renderCell()`

**Comment by ridaamirini** [comment]
@TerryZ i added a fix. Please review

**Comment by TerryZ** [comment]
Fixed in `2.1.0`
